x-pack/plugin/eql/qa/correctness/README.md

```diff
@@ -66,3 +66,22 @@ or
 ./gradlew ':x-pack:plugin:eql:qa:correctness:javaRestTest' --tests "org.elasticsearch.xpack.eql.EsEQLCorrectnessIT.test {<queryNo>}" -Dtests.eql_correctness_debug=true
 ```
 
+### Run an ES node manually and run the tests against it 
+
+If one wants to run an ES node manually (most probably to be able to debug the server), needs to run the following:
+
+```shell script
+./gradlew -p x-pack/plugin/eql/qa/correctness runEqlCorrectnessNode --debug-jvm
```

Comment:
If below you provided an example `gradlew` command without using `-p`, I think it's better to be consistent and either use here the same format as the other command or use `-p` in both places. Not sure of the differences though, but I've been advised lately to stop using the `-p` format in favor of something like `gradlew :x-pack:plugin:eql:qa:correctness:runEqlCorrectness --debug-jvm`